### PR TITLE
Update data-table

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for #855

Before : 
Screenshot 2023-09-08 at 3 04 26 PM
After:
Screenshot 2023-09-08 at 2 56 33 PM
Screenshot 2023-09-08 at 2 57 34 PM
Screenshot 2023-09-08 at 2 58 41 PM

### CODE REVIEW

File: django_project/frontend/utils/data_table.py [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+                ],
+                "open": property.open
+            }
+            for area_available in area_available_values
+        ])
+
+    return Property_reports
+
+
+def sampling_report(queryset: QuerySet, request) -> List:
```
Comment (dimasciput) [inline django_project/frontend/utils/data_table.py:161]: Please add tests for this function

File: django_project/frontend/utils/data_table.py [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+        )
+
+        Sampling_reports.extend([
+            {**common_data, **data} for data in sampling_reports_data
+        ])
+
+    return Sampling_reports
+
+
+def activity_report(queryset: QuerySet, request) -> Dict[str, List[Dict]]:
```
Comment (dimasciput) [inline django_project/frontend/utils/data_table.py:200]: Please add tests for this function

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: django_project/frontend/utils/data_table.py (hunk 1 above)
Comment (ketankartoza) [reply django_project/frontend/utils/data_table.py:161]: Okay

File: django_project/frontend/utils/data_table.py (hunk 2 above)
Comment (ketankartoza) [reply django_project/frontend/utils/data_table.py:200]: Sure

File: django_project/frontend/utils/data_table.py [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+
+
+def species_report(queryset: QuerySet, request) -> List:
+    """
+    Generate species reports based on the user's request.
+    Params:
+        queryset (QuerySet): Properties queryset to generate reports from.
+        request: The HTTP request object.
+    """
+    Species_population_reports = []
```
Comment (dimasciput) [inline django_project/frontend/utils/data_table.py:79]: Please follow python coding standard

File: django_project/frontend/utils/data_table.py [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+
+
+def property_report(queryset: QuerySet, request) -> List:
+    """
+    Generate property reports based on the user's request.
+    Params:
+        queryset (QuerySet): Properties queryset to generate reports from.
+        request: The HTTP request object.
+    """
+    Property_reports = []
```
Comment (dimasciput) [inline django_project/frontend/utils/data_table.py:118]:
```suggestion
    property_reports = []
```

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Review by dimasciput — APPROVED** [review]

**Pull request merged**